### ISSUE TITLE
Consensus stuck in sync mode

### ISSUE THREAD

**Issue by sophoah** [harmony-one/harmony#4534]
**Describe the bug**

During the normal validator node operation, consensus detects that the node is no longer in sync (goes to mode: `syncing` and never completes the consensus phases.

```json
{"level":"info","hostID":"QmYuVNBxs1QtBg1G5GA7LEbBZD5ypm2TCJtV4wQ1yuCUie","node":"QmQsaV5txtYdGgTCrp6GwSmS5ghHSR9dtmwJTfvkFtUrLq","caller":"/root/go/src/github.com/harmony-one/harmony/p2p/host.go:550","time":"2023-10-13T11:33:25.103815026+02:00","message":"peer disconnected"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Announce","mode":"Syncing","MsgViewID":132298,"MsgBlockNum":132297,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:39","time":"2023-10-13T11:33:25.721358915+02:00","message":"[OnAnnounce] Announce message Added"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Announce","mode":"Syncing","blockHash":"1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:154","time":"2023-10-13T11:33:25.72279097+02:00","message":"[OnAnnounce] Sent Prepare Message!!"}
{"level":"info","caller":"/root/go/src/github.com/harmony-one/harmony/internal/chain/reward.go:297","time":"2023-10-13T11:33:25.72660896+02:00","message":"Commit sigs are ready"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:68","time":"2023-10-13T11:33:25.726928569+02:00","message":"[Announce] Block verified"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","MsgBlockNum":132297,"MsgViewID":132298,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:188","time":"2023-10-13T11:33:25.765571879+02:00","message":"[OnPrepared] Received prepared message"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:96","time":"2023-10-13T11:33:25.768788839+02:00","message":"[validateNewBlock] Block Already verified"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:264","time":"2023-10-13T11:33:25.768894902+02:00","message":"[OnPrepared] Not in normal mode, Exiting!!"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","MsgBlockNum":132297,"MsgViewID":132298,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:291","time":"2023-10-13T11:33:25.897055171+02:00","message":"[OnCommitted] Received committed message"}
{"level":"info","myBlock":132297,"myViewID":132298,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:645","time":"2023-10-13T11:33:25.902511433+02:00","message":"[TryCatchup] Adding block to chain"}
{"level":"info","block num":"132297","caller":"/root/go/src/github.com/harmony-one/harmony/core/state_processor.go:115","time":"2023-10-13T11:33:25.90277902+02:00","message":"result cache hit."}
{"level":"info","blockNum":131897,"shardID":1,"caller":"/root/go/src/github.com/harmony-one/harmony/core/offchain.go:165","time":"2023-10-13T11:33:25.903002868+02:00","message":"[insertChain/crosslinks] Cross Link Added to Beaconchain"}
{"level":"info","number":"132297","hash":"0x1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","uncles":0,"txs":0,"stakingTxs":0,"gas":0,"elapsed":"777.455µs","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:1958","time":"2023-10-13T11:33:25.90341318+02:00","message":"Inserted new block"}
{"level":"info","blocks":1,"txs":0,"mgas":0,"elapsed":"838.076µs","mgasps":0,"number":"132297","hash":"0x1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","cache":"1.99 KiB","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:2038","time":"2023-10-13T11:33:25.903466663+02:00","message":"Imported new chain segment"}
{"level":"info","blockNum":132297,"hash":"0x1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:683","time":"2023-10-13T11:33:25.903599987+02:00","message":"Added New Block to Blockchain!!!"}
{"level":"info","myBlock":132298,"myViewID":132299,"phase":"Announce","mode":"Syncing","From":132297,"To":132298,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:798","time":"2023-10-13T11:33:25.903619167+02:00","message":"[TryCatchup] Caught up!"}
{"level":"info","myBlock":132298,"myViewID":132299,"phase":"Announce","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:393","time":"2023-10-13T11:33:25.903660959+02:00","message":"[OnCommitted] Start consensus timer (new block added)"}
{"level":"info","blockNum":132297,"epochNum":146,"ViewId":132298,"blockHash":"0x1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","numTxns":0,"numStakingTxns":0,"numSignatures":0,"caller":"/root/go/src/github.com/harmony-one/harmony/node/node_handler.go:352","time":"2023-10-13T11:33:25.903852763+02:00","message":"BINGO !!! Reached Consensus"}
```

this issue usually happens fixed by itself when epoch changes or when the node is restarted

in theory if the node can already add the last block :
```json
{"level":"info","blockNum":132297,"epochNum":146,"ViewId":132298,"blockHash":"0x1d6c9e84eadb66a570d23a671f62b1a33424fe4a301a24ac14e438729eba6ef1","numTxns":0,"numStakingTxns":0,"numSignatures":0,"caller":"/root/go/src/github.com/harmony-one/harmony/node/node_handler.go:352","time":"2023-10-13T11:33:25.903852763+02:00","message":"BINGO !!! Reached Consensus"}
```
we should change the consensus mode here back to normal. Not sure why the consensus loop doesn't detect it is in sync already.

**To Reproduce**
monitor when the consensus goes to syncing ..

**Expected behavior**
node is already in synced since it is adding new blocks, however node doesn't know until we restart.

**Screenshots**

**Environment (please complete the following information):**
- Harmony v8176-v2023.2.6-66-ge1338060 

**Additional context**

**Comment by sophoah** [comment]
see logs here that shows the last few blocks before epoch change, the consensus mode that switch back to normal and the the correct BINGOs (ie with `numSignature` not at 0)
```json
{"level":"info","myBlock":132299,"myViewID":132300,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:264","time":"2023-10-13T11:33:30.090593428+02:00","message":"[OnPrepared] Not in normal mode, Exiting!!"}
{"level":"info","hostID":"QmYuVNBxs1QtBg1G5GA7LEbBZD5ypm2TCJtV4wQ1yuCUie","node":"QmQjqmir25RnHrQepXnYPzQuWLLpobFggKzYd3GKjLS4bG","caller":"/root/go/src/github.com/harmony-one/harmony/p2p/host.go:550","time":"2023-10-13T11:33:30.117046796+02:00","message":"peer disconnected"}
{"level":"info","myBlock":132299,"myViewID":132300,"phase":"Prepare","mode":"Syncing","MsgBlockNum":132299,"MsgViewID":132300,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:291","time":"2023-10-13T11:33:31.523413344+02:00","message":"[OnCommitted] Received committed message"}
{"level":"info","myBlock":132299,"myViewID":132300,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:645","time":"2023-10-13T11:33:31.528946038+02:00","message":"[TryCatchup] Adding block to chain"}
{"level":"info","block num":"132299","caller":"/root/go/src/github.com/harmony-one/harmony/core/state_processor.go:115","time":"2023-10-13T11:33:31.52936942+02:00","message":"result cache hit."}
{"level":"info","epoch":"147","size":752,"caller":"/root/go/src/github.com/harmony-one/harmony/core/rawdb/accessors_offchain.go:46","time":"2023-10-13T11:33:31.529828095+02:00","message":"wrote sharding state"}
{"level":"info","blockNum":131899,"shardID":1,"caller":"/root/go/src/github.com/harmony-one/harmony/core/offchain.go:165","time":"2023-10-13T11:33:31.529889737+02:00","message":"[insertChain/crosslinks] Cross Link Added to Beaconchain"}
{"level":"info","number":"132299","hash":"0x90a0b3793b6bdd5b7501927243a00f7926671dacc4785ac310be3a817fc58141","uncles":0,"txs":0,"stakingTxs":0,"gas":0,"elapsed":"1.462ms","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:1958","time":"2023-10-13T11:33:31.530562787+02:00","message":"Inserted new block"}
{"level":"info","blocks":1,"txs":0,"mgas":0,"elapsed":"1.525ms","mgasps":0,"number":"132299","hash":"0x90a0b3793b6bdd5b7501927243a00f7926671dacc4785ac310be3a817fc58141","cache":"1.33 KiB","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:2038","time":"2023-10-13T11:33:31.530667032+02:00","message":"Imported new chain segment"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:359","time":"2023-10-13T11:33:31.530844297+02:00","message":"[UpdateConsensusInformation] Updating....."}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","numPubKeys":5,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:411","time":"2023-10-13T11:33:31.530871168+02:00","message":"[UpdateConsensusInformation] Successfully updated public keys"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:86","time":"2023-10-13T11:33:31.530904349+02:00","message":"My Committee updated"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":0,"BLSPubKey":"e7f54994bc5c02edeeb178ce2d34db276a893bab5c59ac3d7eb9f077c893f9e31171de6236ba0e21be415d8631e45b91","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:91","time":"2023-10-13T11:33:31.530920789+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":1,"BLSPubKey":"e212c4c5cd8e078aedd34c300ae500c5bd5fd036231d1c9dcc8400337919a59ebd47523faa209fc0d873ead492621f88","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:91","time":"2023-10-13T11:33:31.53092991+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":2,"BLSPubKey":"99d0835797ca0683fb7b1d14a882879652ddcdcfe0d52385ffddf8012ee804d92e5c05a56c9d7fc663678e36a158a28c","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:91","time":"2023-10-13T11:33:31.53093771+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":3,"BLSPubKey":"9f5f8c5dbe12d14ae8468828754c4db3f2129c69de88baee4e361e38000aed54da204591bbcdeeeb571ffff81e9ade02","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:91","time":"2023-10-13T11:33:31.5309455+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":4,"BLSPubKey":"8a211eb5e9334341fd2498fb5d6b922b4a0984d6a4ea0b5631c1904de5fe21fd6889c9c032d862546ca50a5c41294b0c","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:91","time":"2023-10-13T11:33:31.53096267+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","info":"e7f54994bc5c02edeeb178ce2d34db276a893bab5c59ac3d7eb9f077c893f9e31171de6236ba0e21be415d8631e45b91","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:98","time":"2023-10-13T11:33:31.530973831+02:00","message":"Setting leader as first validator, because provided new keys"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":0,"BLSPubKey":"e7f54994bc5c02edeeb178ce2d34db276a893bab5c59ac3d7eb9f077c893f9e31171de6236ba0e21be415d8631e45b91","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:107","time":"2023-10-13T11:33:31.530992711+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":1,"BLSPubKey":"e212c4c5cd8e078aedd34c300ae500c5bd5fd036231d1c9dcc8400337919a59ebd47523faa209fc0d873ead492621f88","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:107","time":"2023-10-13T11:33:31.531000822+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":2,"BLSPubKey":"99d0835797ca0683fb7b1d14a882879652ddcdcfe0d52385ffddf8012ee804d92e5c05a56c9d7fc663678e36a158a28c","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:107","time":"2023-10-13T11:33:31.531008642+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":3,"BLSPubKey":"9f5f8c5dbe12d14ae8468828754c4db3f2129c69de88baee4e361e38000aed54da204591bbcdeeeb571ffff81e9ade02","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:107","time":"2023-10-13T11:33:31.531017802+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Syncing","index":4,"BLSPubKey":"8a211eb5e9334341fd2498fb5d6b922b4a0984d6a4ea0b5631c1904de5fe21fd6889c9c032d862546ca50a5c41294b0c","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_service.go:107","time":"2023-10-13T11:33:31.531035543+02:00","message":"Member"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Announce","mode":"Syncing","Phase":"Announce","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/view_change.go:579","time":"2023-10-13T11:33:31.531083174+02:00","message":"[ResetViewChangeState] Resetting view change state"}
{"level":"info","blockNum":132299,"hash":"0x90a0b3793b6bdd5b7501927243a00f7926671dacc4785ac310be3a817fc58141","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:683","time":"2023-10-13T11:33:31.533083707+02:00","message":"Added New Block to Blockchain!!!"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Announce","mode":"Normal","From":132299,"To":132300,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:798","time":"2023-10-13T11:33:31.533128199+02:00","message":"[TryCatchup] Caught up!"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Announce","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:393","time":"2023-10-13T11:33:31.533165819+02:00","message":"[OnCommitted] Start consensus timer (new block added)"}
{"level":"info","blockNum":132299,"epochNum":146,"ViewId":132300,"blockHash":"0x90a0b3793b6bdd5b7501927243a00f7926671dacc4785ac310be3a817fc58141","numTxns":0,"numStakingTxns":0,"numSignatures":0,"caller":"/root/go/src/github.com/harmony-one/harmony/node/node_handler.go:352","time":"2023-10-13T11:33:31.533267022+02:00","message":"BINGO !!! Reached Consensus"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Announce","mode":"Normal","MsgViewID":132301,"MsgBlockNum":132300,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:39","time":"2023-10-13T11:33:31.721089638+02:00","message":"[OnAnnounce] Announce message Added"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Announce","mode":"Normal","blockHash":"5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:154","time":"2023-10-13T11:33:31.72212453+02:00","message":"[OnAnnounce] Sent Prepare Message!!"}
{"level":"info","caller":"/root/go/src/github.com/harmony-one/harmony/internal/chain/reward.go:297","time":"2023-10-13T11:33:31.726038532+02:00","message":"Commit sigs are ready"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:68","time":"2023-10-13T11:33:31.726269069+02:00","message":"[Announce] Block verified"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Normal","MsgBlockNum":132300,"MsgViewID":132301,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:188","time":"2023-10-13T11:33:31.7717084+02:00","message":"[OnPrepared] Received prepared message"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:96","time":"2023-10-13T11:33:31.774610061+02:00","message":"[validateNewBlock] Block Already verified"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Prepare","mode":"Normal","blockNum":132300,"blockHash":"5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:178","time":"2023-10-13T11:33:31.775588962+02:00","message":"[sendCommitMessages] Sent Commit Message!!"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Commit","mode":"Normal","MsgBlockNum":132300,"MsgViewID":132301,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:291","time":"2023-10-13T11:33:31.799884003+02:00","message":"[OnCommitted] Received committed message"}
{"level":"info","myBlock":132300,"myViewID":132301,"phase":"Commit","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:645","time":"2023-10-13T11:33:31.804155956+02:00","message":"[TryCatchup] Adding block to chain"}
{"level":"info","block num":"132300","caller":"/root/go/src/github.com/harmony-one/harmony/core/state_processor.go:115","time":"2023-10-13T11:33:31.804479345+02:00","message":"result cache hit."}
{"level":"info","blockNum":131900,"shardID":1,"caller":"/root/go/src/github.com/harmony-one/harmony/core/offchain.go:165","time":"2023-10-13T11:33:31.804763594+02:00","message":"[insertChain/crosslinks] Cross Link Added to Beaconchain"}
{"level":"info","blockNum":131901,"shardID":1,"caller":"/root/go/src/github.com/harmony-one/harmony/core/offchain.go:165","time":"2023-10-13T11:33:31.804788256+02:00","message":"[insertChain/crosslinks] Cross Link Added to Beaconchain"}
{"level":"info","number":"132300","hash":"0x5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","uncles":0,"txs":0,"stakingTxs":0,"gas":0,"elapsed":"931.939µs","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:1958","time":"2023-10-13T11:33:31.805176777+02:00","message":"Inserted new block"}
{"level":"info","blocks":1,"txs":0,"mgas":0,"elapsed":"981.031µs","mgasps":0,"number":"132300","hash":"0x5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","cache":"1.33 KiB","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:2038","time":"2023-10-13T11:33:31.805230609+02:00","message":"Imported new chain segment"}
{"level":"info","blockNum":132300,"hash":"0x5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:683","time":"2023-10-13T11:33:31.805360023+02:00","message":"Added New Block to Blockchain!!!"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Announce","mode":"Normal","From":132300,"To":132301,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:798","time":"2023-10-13T11:33:31.805372703+02:00","message":"[TryCatchup] Caught up!"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Announce","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:393","time":"2023-10-13T11:33:31.805389234+02:00","message":"[OnCommitted] Start consensus timer (new block added)"}
{"level":"info","blockNum":132300,"epochNum":147,"ViewId":132301,"blockHash":"0x5105636745bab5f40b5cda283892acc2b57dd50b53eeae521cf0813298d5bd6b","numTxns":0,"numStakingTxns":0,"numSignatures":0,"caller":"/root/go/src/github.com/harmony-one/harmony/node/node_handler.go:352","time":"2023-10-13T11:33:31.805508307+02:00","message":"BINGO !!! Reached Consensus"}
{"level":"info","hostID":"QmYuVNBxs1QtBg1G5GA7LEbBZD5ypm2TCJtV4wQ1yuCUie","node":"QmQjqmir25RnHrQepXnYPzQuWLLpobFggKzYd3GKjLS4bG","caller":"/root/go/src/github.com/harmony-one/harmony/p2p/host.go:539","time":"2023-10-13T11:33:32.308914313+02:00","message":"peer connected"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Announce","mode":"Normal","MsgViewID":132302,"MsgBlockNum":132301,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:39","time":"2023-10-13T11:33:33.721869554+02:00","message":"[OnAnnounce] Announce message Added"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Announce","mode":"Normal","blockHash":"f035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:154","time":"2023-10-13T11:33:33.723114604+02:00","message":"[OnAnnounce] Sent Prepare Message!!"}
{"level":"info","caller":"/root/go/src/github.com/harmony-one/harmony/internal/chain/reward.go:297","time":"2023-10-13T11:33:33.731123564+02:00","message":"Commit sigs are ready"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Prepare","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:68","time":"2023-10-13T11:33:33.731285688+02:00","message":"[Announce] Block verified"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Prepare","mode":"Normal","MsgBlockNum":132301,"MsgViewID":132302,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:188","time":"2023-10-13T11:33:33.756505558+02:00","message":"[OnPrepared] Received prepared message"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Prepare","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:96","time":"2023-10-13T11:33:33.760795042+02:00","message":"[validateNewBlock] Block Already verified"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Prepare","mode":"Normal","blockNum":132301,"blockHash":"f035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:178","time":"2023-10-13T11:33:33.762120433+02:00","message":"[sendCommitMessages] Sent Commit Message!!"}
{"level":"info","hostID":"QmYuVNBxs1QtBg1G5GA7LEbBZD5ypm2TCJtV4wQ1yuCUie","node":"QmNaM2wdJGz1pg3Qg4TEoniNA5YbgbD9Y3G2qNFRVkNYMJ","caller":"/root/go/src/github.com/harmony-one/harmony/p2p/host.go:550","time":"2023-10-13T11:33:33.792981949+02:00","message":"peer disconnected"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Commit","mode":"Normal","MsgBlockNum":132301,"MsgViewID":132302,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:291","time":"2023-10-13T11:33:33.793026209+02:00","message":"[OnCommitted] Received committed message"}
{"level":"info","myBlock":132301,"myViewID":132302,"phase":"Commit","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:645","time":"2023-10-13T11:33:33.79718599+02:00","message":"[TryCatchup] Adding block to chain"}
{"level":"info","block num":"132301","caller":"/root/go/src/github.com/harmony-one/harmony/core/state_processor.go:115","time":"2023-10-13T11:33:33.797402177+02:00","message":"result cache hit."}
{"level":"info","number":"132301","hash":"0xf035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","uncles":0,"txs":0,"stakingTxs":0,"gas":0,"elapsed":"525.446µs","caller":"/roo
t/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:1958","time":"2023-10-13T11:33:33.797783689+02:00","message":"Inserted new block"}
{"level":"info","blocks":1,"txs":0,"mgas":0,"elapsed":"561.598µs","mgasps":0,"number":"132301","hash":"0xf035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","cache":"1.33 KiB","caller":"/root/go/src/github.com/harmony-one/harmony/core/blockchain_impl.go:2038","time":"2023-10-13T11:33:33.79781962+02:00","message":"Imported new chain segment"}
{"level":"info","blockNum":132301,"hash":"0xf035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:683","time":"2023-10-13T11:33:33.797933434+02:00","message":"Added New Block to Blockchain!!!"}
{"level":"info","myBlock":132302,"myViewID":132303,"phase":"Announce","mode":"Normal","From":132301,"To":132302,"caller":"/root/go/src/github.com/harmony-one/harmony/consensus/consensus_v2.go:798","time":"2023-10-13T11:33:33.797949934+02:00","message":"[TryCatchup] Caught up!"}
{"level":"info","myBlock":132302,"myViewID":132303,"phase":"Announce","mode":"Normal","caller":"/root/go/src/github.com/harmony-one/harmony/consensus/validator.go:393","time":"2023-10-13T11:33:33.797968535+02:00","message":"[OnCommitted] Start consensus timer (new block added)"}
{"level":"info","blockNum":132301,"epochNum":147,"ViewId":132302,"blockHash":"0xf035b8cf8defdbe0a4a45f57ce53ceb72cb8d0a73c8e9a7dca8511ed7e8264b4","numTxns":0,"numStakingTxns":0,"numSignatures":1,"caller":"/root/go/src/github.com/harmony-one/harmony/node/node_handler.go:352","time":"2023-10-13T11:33:33.798040577+02:00","message":"BINGO !!! Reached Consensus"}
```

**Comment by sophoah** [comment]
fix in #4542
